src/macro-carousel/macro-carousel.js

```diff
@@ -480,7 +486,7 @@ class MacroCarousel extends HTMLElement {
    * binding every function. See
    * https://medium.com/@WebReflection/dom-handleevent-a-cross-platform-standard-since-year-2000-5bf17287fd38
    *
-   * @param {Event} e Any event.
+   * @param {Event|KeyboardEvent} e Any event.
```

Comment:
I think `KeyboardEvent` is a subclass of `Event`, is there any specific reason why you want to specify it?

Reply:
Mainly to satisfy IDE's, not a biggie - will revert